tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -340,8 +340,13 @@ def _transpose_handler(self, trans, node):
             ops = self._g.get_nodes()
             self._g.replace_all_inputs(ops, node.output[0], trans.input[0])
 
+            shape = self._g.get_shape(node.output[0])
+            dtype = self._g.get_dtype(node.output[0])

```

Comment:
please add tests to cover these cases.

Reply:
1 transpose node is graph output has been covered by test_transpose_leaky_relu
2 identity is graph output is covered by new-added test case "test_transpose_with_identity"